Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM metabrainz/python:3.7-20190302 as listenbrainz-base
+FROM metabrainz/python:newconsul as listenbrainz-base
```

Comment:
I just built this image locally for testing - https://github.com/metabrainz/docker-python/blob/master/3.7/Dockerfile#L1
Change to `docker/beta/timescale_writer/consul-template-timescale-writeFROM metabrainz/consul-template-base:v0.18.5-2` and `docker build -t metabrainz/python:newconsul .`